benchmark/common.js

```diff
@@ -1,17 +1,17 @@
 'use strict';
 
 const child_process = require('child_process');
-
-// The port used by servers and wrk
-exports.PORT = process.env.PORT || 12346;
+const http_benchmarkers = require('./_http-benchmarkers.js');
 
 exports.createBenchmark = function(fn, options) {
   return new Benchmark(fn, options);
 };
 
 function Benchmark(fn, options) {
   this.name = require.main.filename.slice(__dirname.length + 1);
-  this.options = this._parseArgs(process.argv.slice(2), options);
+  const parsed_args = this._parseArgs(process.argv.slice(2), options);
+  this.options = parsed_args.cli;
+  this.extra_options = parsed_args.extra;
```

Comment:
If we just add those to options they will be displayed when running other benchmarks, even unrelated ones. It does not look, and could be confusing. So I store those extra options elsewhere and apply them only to related benchmarks